test/catalogue.test.js

```diff
@@ -239,68 +263,104 @@ describe('Catalogue', () => {
         it('should replace identifiers with pure MAQL', () => {
             const mockPayload = {
```

Comment:
You might consider moving these large objects to a new fixtures file.

Reply:
moved to existing fixtures file